docs/en/stack/ml/df-analytics/dfa-regression.asciidoc

```diff
@@ -0,0 +1,103 @@
+[role="xpack"]
+[[dfa-regression]]
+== {regression-cap}
+
+{reganalysis-cap} is a {ml} process for estimating the relationships among 
+different variables, then making further predictions based on these 
+relationships.
+
+For example, suppose we are interested in finding the relationship between 
+apartment size and monthly rent in a city. Our imaginary dataset consists of 
+three data points:
+
+|===
+| Size (m2) | Monthly rent 
+| 44        | 1600
+| 24        | 1055
+| 63        | 2300
+|===
+
+After the model determines the relationship between the {feature-vars} 
```

Comment:
Since we haven't mentioned feature variables yet at this point, I think we can omit it here too:
```suggestion
After the model determines the relationship between the
```